relentless/simulate/hoomd.py

```diff
@@ -593,10 +621,12 @@ def __init__(self, check_thermo_every, check_rdf_every, rdf_dr):
     def __call__(self, sim):
         with sim.context:
             # thermodynamic properties
+            if isinstance(sim.ensemble.V, Volume):
+                quantities_logged = ['temperature','pressure','lx','ly','lz','xy','xz','yz']
+            if isinstance(sim.ensemble.V, Area):
```

Comment:
Add an `else` case to error out if neither of these is found
```suggestion
            elif isinstance(sim.ensemble.V, Area):
```